loops.js

```diff
@@ -51,3 +69,8 @@
 // >> 9
 // ==========================================
 
+for (let i =0; i < 10; i++) {
```

Comment:
Code is lekker kort en werkt goed! 
Alleen voor de leesbaarheid moet je wel na elke **if statement** de accolades plaatsen.
for (let i =0; i < 10; i++) {
    if (i < 3) { 
    console.log(i); 
    }
    if (i > 2 && i < 6) { 
    console.log("> " + i); 
    }
    if (i > 5 ) {
    console.log(">> " + i);
    }
}